blockchain/fastsync.go

```diff
@@ -213,38 +213,69 @@ func worker(cache *utxoCache, jobs <-chan []byte, results chan<- *result) {
 // downloadUtxoSet will attempt to connect to make an HTTP GET request to the
 // provided sources one at a time and download the UTXO set to the provided path.
 // If a proxy is provided it will use it for the HTTP connection.
-func downloadUtxoSet(sources []string, proxy *socks.Proxy, pth string) (string, error) {
-	var fileName string
-
-	grab.SetDefaultClientWithProxy(proxy)
-
-	for _, src := range sources {
-		log.Infof("Downloading UTXO set from %s", src)
-		err := retry.Do(
-			func() error {
-				resp, err := grab.Get(pth, src)
-				if err != nil {
-					return err
+func downloadUtxoSet(sources []string, proxy *socks.Proxy, destination string) (string, error) {
+	for _, source := range sources {
+		log.Infof("FastSync: Downloading UTXO set from %s to %s ...", source, destination)
```

Comment:
I think we need a trailing `\n` here too.

Reply:
Actually NM, this isn't needed. I just checked and Infof has different semantics than printf.